modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding the {product-title} container images
+You can embed {product-title}'s container images into the {op-system-ostree} images, so that they are immediately available to the CRI-O container engine after booting, without having to pull the images over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
+[source,terminal]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+To embed {product-title}'s container images, you need to know the exact list of container image references used by the {product-title} version you are embedding. You can get this list from the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint using the following procedure.
+
+You need to have the exact list of container image references used by the {product-title} version to embed {product-title} 's container images. Use the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint.
+
+.Procedure
+
+. Download the `microshift-release-info` RPM package matching your {product-title} version running the following command:
++
+[source,terminal]
+----
+$ yum download microshift-release-info-4.12.0-1.el8
+----
```

Comment:
I would automate current version extraction using the following command, and then use it in the subsequent listings.
```
VERSION=$(rpm -q --queryformat='%{VERSION}-%{RELEASE}.%{ARCH}' microshift)
```

Reply:
It would be something like this:
```
$ VERSION=$(rpm -q --queryformat='%{VERSION}-%{RELEASE}.%{ARCH}' microshift)
$ yum download microshift-release-info-${VERSION}
$ rpm2cpio microshift-release-info-${VERSION} | cpio -idmv
```

Reply:
In other places we use attributes to inject version numbers into commands. That makes the commands easier for users to understand. Can we do that here?

Reply:
We can use the [source,terminal,subs="attributes+"] and use attributes, sure. Where should we substitute? `microshift-release-info` is the name of the RPM package in the $ yum, so do not know what we are changing that to. @ggiguash 's three suggestions? Where would attributes go there? I don't want to get anything wrong here, so please write out which commands we know work.

Reply:
Looking again, I see that the RPM version number is going to change a lot more often than the docs. Let's go with @ggiguash 's proposal of looking up the value and setting the variable.

Reply:
@dhellmann  See the latest commit: 5789da6. PTAL and tell me if this needs a correction.